src/apscheduler/datastores/sqlalchemy.py

```diff
@@ -231,6 +232,12 @@ async def _begin_transaction(
 
             yield conn
 
+    async def _create_schema(self, conn: Connection | AsyncConnection) -> None:
+        if not self.schema:
+            return
+        t = schema.CreateSchema(name=self.schema, if_not_exists=True)
```

Comment:
I'd prefer that you imported this like `from sqlalchemy.schema import CreateSchema`.